### PR TITLE
Add faq for retrieving profile

### PR DESCRIPTION
**Pull request by noahpc**

### CODE REVIEW

**Review by singhald — APPROVED** [review]

File: lightning-tag/faq.md
Comment (ctmitchell28) [inline lightning-tag/faq.md:79]: This looks good. Can we reference this in the Google Ad Manager page somewhere? 

"You may also need to push the Lotame profile Id into the Google Ad Manager call (ex. this is required before you can send Lotame your Google Ad Log files for automated campaign analytics). If you need to pass the Lotame profile Id into the Google Ad Manager ad calls, please reference <link>"

**Review by ctmitchell28 — APPROVED** [review]
This looks good. Can we reference this in the Google Ad Manager page somewhere?

"You may also need to push the Lotame profile Id into the Google Ad Manager call (ex. this is required before you can send Lotame your Google Ad Log files for automated campaign analytics). If you need to pass the Lotame profile Id into the Google Ad Manager ad calls, please reference "

File: lightning-tag/faq.md
Comment (noahpc) [reply lightning-tag/faq.md:79]: Once I get my other prs merged in, I will update that page (since i have other changes on there already, it is easier to wait till those are merged).

**Review by ErikMHummel — APPROVED** [review]

**Review by ctmitchell28 — COMMENTED** [review]
Looks good

**Pull request merged**
